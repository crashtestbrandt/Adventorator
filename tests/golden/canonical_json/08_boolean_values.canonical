{"false_val":false,"number":42,"true_val":true}