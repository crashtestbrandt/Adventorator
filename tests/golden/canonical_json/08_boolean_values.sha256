0f5dd641cdcb7ee8edf41a34228ec7b077bfd6ffa9a03345b1c75584b07f9bc7
