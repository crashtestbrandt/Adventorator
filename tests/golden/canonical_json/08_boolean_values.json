{
  "true_val": true,
  "false_val": false,
  "number": 42
}